src/content/structured/components/page-header/code.mdx

```diff
@@ -63,7 +64,7 @@ export const withBreadcrumbNavigation = [
   {
     language: "Web component",
     snippet: `<ic-page-header heading="Coffee recipes" subheading="This is a page header component that has breadcrumb navigation.">
-  <ic-status-tag slot="heading-adornment" label="Beta"></ic-status-tag>
+  <ic-chip slot="heading-adornment" label="Beta"></ic-chip>
```

Comment:
```suggestion
  <ic-chip slot="heading-adornment" label="BETA", size="large"></ic-chip>
```
Wherever there is a chip , can we make it use the label "BETA" (uppercase) and use the "large" sized chip